src/packages/fetch-engine/src/download.ts

```diff
@@ -1,27 +1,24 @@
-import fs from 'fs'
-import { promisify } from 'util'
-import chalk from 'chalk'
-
-// Packages
-import path from 'path'
 import Debug from '@prisma/debug'
-import makeDir from 'make-dir'
+import { getos, getPlatform, Platform, platforms } from '@prisma/get-platform'
+import chalk from 'chalk'
 import execa from 'execa'
+import fs from 'fs'
+import makeDir from 'make-dir'
 import pFilter from 'p-filter'
-import hasha from 'hasha'
+// Packages
```

Comment:
If you reorder imports, then please put the `// Packages` to the top

Reply:
prettier